coala_quickstart/info_extractors/RequirementsInfoExtractor.py

```diff
@@ -0,0 +1,33 @@
+from requirements import parse
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    ProjectDependencyInfo, VersionInfo)
+
+
+class RequirementsInfoExtractor(InfoExtractor):
+    supported_file_globs = ("requirements.txt",)
+
+    supported_information_kinds = (
+        ProjectDependencyInfo, VersionInfo)
+
+    def parse_file(self, fname, file_content):
+        parsed_file = []
+        with open(fname, 'r') as f:
+            parsed_file = parse(f)
+        return parsed_file
+
+    def find_information(self, fname, parsed_file):
+        results = []
+        for dependency in parsed_file:
+            results.append(
+                ProjectDependencyInfo(
+                    fname,
+                    dependency.name,
+                    version=VersionInfo(fname,
+                                        ''.join(str(i) for i in
+                                                dependency.specs[0]))
+                )
```

Comment:
I have a suggestion, we could change the `VersionInfo` to either store a list instead of string, or we could add an optional `other_versions` parameters.